spec/models/task_spec.rb

```diff
@@ -1,5 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+	it "タイトルとステータスがあれば有効なこと" do
+		user = create(:user)
+		task = build(:task, user_id: user.id)
+		expect(task).to be_valid
+	end
```

Comment:
Tabでインデントされてしまっているので、`半角スペース2`でインデントするようにしてください。
また、各テストケースの間には1行空行を開けた方が見やすいですね。
```suggestion
	it "タイトルとステータスがあれば有効なこと" do
	  user = create(:user)
	  task = build(:task, user_id: user.id)
	  expect(task).to be_valid
	end

```

Reply:
a31300a で修正しました

Reply:
36e7e67 でテストコードの下に１行開けました